src/mame/machine/bublbobl.cpp

```diff
@@ -35,49 +62,79 @@ WRITE8_MEMBER(bublbobl_state::bublbobl_bankswitch_w)
 	flip_screen_set(data & 0x80);
 }
 
+/* tokio bankswitch reg
+   76543210
+   |||||\\\- Select ROM bank
+   ||||\---- ? used (idle high, /SEQRES?)
+   |||\----- not used?
+   ||\------ not used?
+   |\------- ? used (idle high, /BLACK?)
+   \-------- ? used (idle high, /SRESET?)
+// bublboblp: test and main: 00 C8 C9 C8 C9...; tokio: test 00 09 09 49 main 00 09 C8 CF
+*/
 WRITE8_MEMBER(bublbobl_state::tokio_bankswitch_w)
 {
+	m_screen->update_now();
 	/* bits 0-2 select ROM bank */
 	membank("bank1")->set_entry(data & 7);
 
-	/* bits 3-7 unknown */
+	/* bit 3 unknown */
+	/* GUESS: bit 3 resets mcu */
+	if (m_mcu != nullptr) // only if we have a MCU
+		m_mcu->set_input_line(INPUT_LINE_RESET, (data & 0x08) ? CLEAR_LINE : ASSERT_LINE);
+
+	/* bit 4 and 5 unknown, not used? */
+
+	/* bit 6 is unknown */
+	/* GUESS: bit 6 is video enable "/BLACK" */
+	m_video_enable = data & 0x40; // guess
+
+	/* bit 7 is unknown */
+	m_subcpu->set_input_line(INPUT_LINE_RESET, (data & 0x80) ? CLEAR_LINE : ASSERT_LINE);
```

Comment:
No, I actually left that in by accident. Until we find tokio schematics or someone traces the pcb out, the meaning of the bits isn't known. However, m_video_enable is strongly suspected as it gets set seemingly the same time the bublbobl code sets it in the later hw.